ui/raidboss/data/triggers/o2s.js

```diff
@@ -16,93 +17,202 @@
     },
     {
       id: 'O2S Levitation',
-      regex: /:(\y{Name}) gains the effect of (?:Unknown_556|Levitation) from/,
+      regex: /:(\y{Name}) gains the effect of Levitation from/,
       condition: function(data, matches) { return matches[1] == data.me; },
       run: function(data) { data.levitating = true; },
     },
     {
       id: 'O2S Levitation',
-      regex: /:(\y{Name}) loses the effect of (?:Unknown_556|Levitation)/,
+      regex: /:(\y{Name}) loses the effect of Levitation/,
```

Comment:
Did you miss a regexDe here too?

Reply:
Nope, it's also Levitation in German, so this doesn't need a RegexDe :>